library/src/main/java/com/proxerme/library/connection/info/request/EntryCoreRequest.java

```diff
@@ -0,0 +1,62 @@
+package com.proxerme.library.connection.info.request;
+
+import android.support.annotation.IntRange;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+
+import com.afollestad.bridge.Form;
+import com.afollestad.bridge.Response;
+import com.proxerme.library.connection.ProxerRequest;
+import com.proxerme.library.connection.info.result.EntryCoreResult;
+import com.proxerme.library.info.ProxerTag;
+import com.proxerme.library.info.ProxerUrlHolder;
+
+/**
+ * Request object for the core data of an entry by an id (Anime or Manga).
+ *
+ * @author Desnoo
+ */
+public class EntryCoreRequest extends ProxerRequest<EntryCoreResult> {
+
+    private static final String ENTRY_CORE_URL = "/api/v1/info/entry";
+
+    private static final String ENTRY_CORE_ID = "id";
+
+    /**
+     * The entry id to request.
+     */
```

Comment:
Private properties do not need to be annotated with JavaDoc as they are not part of the public API. In my opinion this makes the code less readable.